### ISSUE TITLE
Print more than one error

### ISSUE THREAD

**Issue by davewongillies** [wata727/tflint#299]
When `tflint` finds errors in Terraform code, it only prints out a single error:

```
$ tflint
Error: Failed to load configurations: main.tf:39,51-74: Missing newline after argument; An argument definition must end with a newline., and 41 other diagnostic(s)
```

It'd be really helpful for it to print out all the errors it finds

**Comment by wata727** [comment]
All multiple errors are reported by Terraform, so you can probably get the equivalent complete errors from `terraform validate`. Errors specific to TFLint are not omitted. For that reason, I decided to omit these errors in TFLint.

If it is helpful to print out all the errors as well as Terraform, probably, it will be able to output multiple context-rich errors om the same way as Terraform.
